CHANGES.md

```diff
@@ -49,9 +49,27 @@
 # [2.32.X] - Unreleased
 
 ## Highlights
-
 * New highly anticipated feature X added to Python SDK ([BEAM-X](https://issues.apache.org/jira/browse/BEAM-X)).
 * New highly anticipated feature Y added to Java SDK ([BEAM-Y](https://issues.apache.org/jira/browse/BEAM-Y)).
+* The [Beam DataFrame
+  API](https://beam.apache.org/documentation/dsls/dataframes/overview/) is no
+  longer experimental! We've spent the time since the [2.32.0 preview
+  announcement](https://beam.apache.org/blog/dataframe-api-preview-available/)
+  implementing the most frequently used pandas operations
```

Comment:
Capitalize "pandas"

Reply:
I think lowercase is preferred here, e.g. it's uncapitalized everywhere it's referenced on http://pandas.pydata.org

Reply:
Oh gotcha, didn't notice that.